IaC/PnP/monitoring/aks/online/aks_secure_baseline/ngsa-memory/ngsa-memory.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    name: ngsa
+  name: ngsa
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ngsa-memory
+  namespace: ngsa
+# We should label our app with proper k8s label
+  labels:
+    app.kubernetes.io/name: ngsa-memory
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: ngsa-memory
+  template:
+    metadata:
+      labels:
+        app: ngsa-memory
+        version: beta-0329-1641
+        deploy: 21-03-29-16-41-59
+      annotations:
+        prometheus.io/scrape: 'true'
```

Comment:
Is the port correct? Should it be 8080? /metrics is on 8080.

Reply:
I thought so too, but apparently, the default port is 9102, even if the default path is /metrics. If NGSA is not using the default port, this can be overwritten. I'll add a note in the code

[Reference](https://developer.ibm.com/technologies/containers/tutorials/monitoring-kubernetes-prometheus/)